spec/spec.md

```diff
@@ -778,13 +778,19 @@ encoding format, we recommend additional considerations to keep payload sizes mi
 
 #### Representing Keys
 
-Apart from the encoding of the cryptographic key itself, which cannot be further minimized in size, we ****RECOMMENDED****
-the following representations of keys and their identifiers using `JsonWebKey`:
+Apart from the encoding of the cryptographic key itself, which cannot be further minimized in size, it is
+****REQUIRED**** to follow the following guidance on representations of keys and their identifiers 
+using the `JsonWebKey` type:
 
 - The [[ref:Identity Key]]'s identifier ****MUST**** always be `0`.
+
 - Key identifiers (`kid`s) ****MAY**** be omitted. If omitted, upon reconstruction of a DID Document, the JWK `kid`
-is set to its JWK Thumbprint [[spec:RFC7638]].
-- DID Document representations ****SHOULD**** always use fully qualified identifiers (e.g. `did:dht:uodqi99wuzxsz6yx445zxkp8ddwj9q54ocbcg8yifsqru45x63kj#0` as opposed to `0` or `#0`)
+and its corresponding Verification Method `id` is to be set to the key's JWK Thumbprint [[spec:RFC7638]].
+
+- A Verification Method's `id` ****MUST**** always match the corresponding JWK's `kid` property.
+
+- DID Document representations ****SHOULD**** always use fully qualified identifiers (e.g. 
+`did:dht:uodqi99wuzxsz6yx445zxkp8ddwj9q54ocbcg8yifsqru45x63kj#0` as opposed to `0` or `#0`)
```

Comment:
@decentralgabe Can you rebase this section on the current main?  It conflicts with changes previously made in PR #117